src/Compilers/Core/Portable/SymbolDisplay/SymbolDisplayFormat.cs

```diff
@@ -368,7 +379,39 @@ public class SymbolDisplayFormat
             SymbolDisplayPropertyStyle propertyStyle = default(SymbolDisplayPropertyStyle),
             SymbolDisplayLocalOptions localOptions = default(SymbolDisplayLocalOptions),
             SymbolDisplayKindOptions kindOptions = default(SymbolDisplayKindOptions),
-            SymbolDisplayMiscellaneousOptions miscellaneousOptions = default(SymbolDisplayMiscellaneousOptions))
+            SymbolDisplayMiscellaneousOptions miscellaneousOptions = default(SymbolDisplayMiscellaneousOptions),
+            SymbolDisplayConstantValueOptions constantValueOptions = default)
```

Comment:
can you use 'default' consistently here?  either all 'default', or all 'default(...)'